File: basic_examples/3_Working_with_geometry/ex3_2_parameters_and_variations/materials.py

from gemc_api_materials import *


def define_materials(configuration):
	
	# Scintillator
	scintillator = MyMaterial(name="scintillator", description="ctof scintillator material", density="1.032",
		ncomponents="2",  components="C 9 H 10")
	print_mat(configuration, scintillator);

File: clas12/alert/ATOF_geom/materials.pl
use strict;
use warnings;

our %configuration;


sub materials
{
	# uploading the mat definition
	
	# ALERT time of flight scintillator
	my %mat = init_mat();
	$mat{"name"}          = "scintillator";
	$mat{"description"}   = "scintillator material taken from ftof";
	$mat{"density"}       = "1.032";
	$mat{"ncomponents"}   = "2";
	$mat{"components"}    = "C 9 H 10";
	print_mat(\%configuration, \%mat);
	
}
